pkg/npm/npm_test.go

```diff
@@ -269,6 +269,28 @@ func TestNpm(t *testing.T) {
 		}
 	})
 
+	t.Run("check Execute all scripts with buildDescriptorList", func(t *testing.T) {
+		utils := newNpmMockUtilsBundle()
+		utils.AddFile("package.json", []byte("{\"scripts\": { \"ci-build\": \"exit 0\" } }"))                       // is filtered out
+		utils.AddFile(filepath.Join("src", "package.json"), []byte("{\"scripts\": { \"ci-build\": \"exit 0\" } }")) // should NOT be filtered out
+
+		options := ExecutorOptions{}
+		runScripts := []string{"ci-build"}
+		buildDescriptorList := []string{filepath.Join("src", "package.json")}
+
+		exec := &Execute{
+			Utils:   &utils,
+			Options: options,
+		}
+		err := exec.RunScriptsInAllPackages(runScripts, nil, nil, false, nil, buildDescriptorList)
+
+		if assert.NoError(t, err) {
+			if assert.Equal(t, 2, len(utils.execRunner.Calls)) {
+				assert.Equal(t, mock.ExecCall{Exec: "npm", Params: []string{"run", "ci-build"}}, utils.execRunner.Calls[1])
```

Comment:
Can you also assert the execution directory here?

Reply:
I don't really have an idea on how to assert the execution directory but I changed the test case. Now it is clear for which of the two modules the call was executed and it is ensured with the assertion that only one script is executed.

Reply:
Sure!